roles/sosreport/defaults/main.yml

```diff
@@ -2,36 +2,10 @@
 sosreport_unarchive: false
 sosreport_tmpdir: /tmp/sosreport
 sosreport_cmd: sosreport
-sosreport_required_packages:
-  - sosreport
+sosreport_required_packages: ""
 sosreport_name: "{{ inventory_hostname_short }}-{{ ansible_date_time.date }}"
 sosreport_archive_filename: "{{ sosreport_name }}.tar.gz"
 sosreport_archive_directory: "/archive/sosreport/{{ ansible_date_time.date }}"
-sosreport_plugins:
-  - apt
-  - auditd
-  - block
-  - devices
-  - docker
-  - dpkg
-  - filesys
-  - hardware
-  - kernel
-  - kvm
-  - md
-  - memory
-  - networking
-  - pci
-  - process
-  - processor
-  - python
-  - services
-  - ssh
-  - system
-  - systemd
-  - ubuntu
-  - udev
-  - usb
-  - xfs
+sosreport_plugins: ""
```

Comment:
not sure if we need this if this is present in /vars

Reply:
probably not